app/src/main/java/com/mmm/parq/fragments/DriverHomeFragment.java

```diff
@@ -71,11 +69,9 @@
     static private final int COARSE_LOCATION_PERMISSION_REQUEST_CODE = 1;
     static private final int FINE_LOCATION_PERMISSION_REQUEST_CODE = 0;
     static private final int LINE_WIDTH = 20;
-    static private final int MAPS_REQUEST_CODE = 1;
     static private final int ZOOM_LEVEL = 16;
 
     private static final String TAG = DriverHomeFragment.class.getSimpleName();
-    private static String CLASS = "DriverHomeFragment";
 
     public enum State {
         FIND_SPOT, NAVIGATION, ARRIVE_SPOT, OCCUPY_SPOT, END_RESERVATION
```

Comment:
ya because the fragments sorta act as the action, not the state itself. I think matching what the fragment does is more important